libvirt/disk/disk_definition.go

```diff
@@ -0,0 +1,22 @@
+package disk
```

Comment:
also this package need to be removed

Reply:
TIL. Thanks!

Will remove both.
I was looking at the gist and saw `disk_definition.go` had been created but was not being used.
Whilst trying to ponder how that would work I created a minimal `volume` equivalent but didn't end up plumbing it in either, figuring can come back to it.